### PR TITLE
Export slider variable for external use

### LINKED ISSUE #55: Make outer slider div bindable
For forms validation, it's important that required but empty inputs can be scrolled into view when the user tries form submission.

Would be great if `svelte-range-slider-pips` exposed the `slider` variable for this purpose so people can call

```js
slider.scrollIntoView()
```

### PR DESCRIPTION
**Pull request by janosh**
Closes #55.

### CODE REVIEW

File: src/RangeSlider.svelte [hunk 1]
```diff
@@ -39,7 +39,7 @@
   const dispatch = createEventDispatcher();
 
   // dom references
-  let slider;
+  export let slider;
```
Comment (simeydotme) [inline src/RangeSlider.svelte:42]: well that was easy 😅

File: src/index.js [hunk 2]
```diff
@@ -1 +1 @@
-export { default as default } from "./RangeSlider.svelte";
\ No newline at end of file
+export { default } from "./RangeSlider.svelte";
```
Comment (simeydotme) [inline src/index.js:1]: while I see this seems to be correct, I have two comments;
1) it's unrelated to the issue
2) this component is cloned from the official component template; https://github.com/sveltejs/component-template/blob/master/src/index.js which uses `{ default as default }` which makes me wonder why they did that?

Comment (janosh) [reply src/index.js:1]: Unrelated indeed. Happy to leave it as is or submit a separate PR if you prefer.

I can only assume whoever wrote the component template didn't know the short version is allowed and that the two are equivalent. I can vouch for it working though. I use it in all my own packages. See e.g. https://github.com/janosh/svelte-multiselect/blob/main/src/lib/index.ts.

**Comment by janosh** [comment]
@simeydotme Any verdict here yet? Would be cool to get this in as it's blocking a new deploy of https://github.com/sbsev/svelte-site.

**Comment by simeydotme** [comment]
Done.

But Janosh, please consider that you have multiple workarounds for this (bind to a hidden span, scroll that into view... or use your own fork until patched...). I have things to do and users to consider, so I need to be happy with the code which is a bit more involved than just looking at the PR.

I get that workarounds are not the ideal solution, and you probably didn't mean to sound pushy. But try something like this in future :) 

> _"hey, just checking if any news on this? Should I consider to apply a workaround in the short-term?"_ 


Hope your build is OK, now.

**Comment by janosh** [comment]
@simeydotme Thanks! My apologies. Did not intend to be pushy. Would have been alright if you said no to this PR.

**Pull request merged**